framework/db/transaction_manager.py

```diff
@@ -438,6 +441,43 @@ def GetByIDAsDict(self, trans_id, target_id=None):
         return self.DeriveTransactionDict(transaction_obj, include_raw_data=True)
 
     @target_required
+    def GetHrtResponse(self, filter_data, trans_id, target_id=None):
+        transaction_obj = self.db.session.query(models.Transaction).filter_by(target_id=target_id, id=trans_id).first()
+
+        # Data validation
+        languages = ['bash']  # Default script language is set to bash.
+        if filter_data.get('language'):
+            languages = map(lambda x: x.strip(), filter_data['language'][0].split(','))
```

Comment:
From POST request of UI, I receives the list of lists of post data.
filter_data = dict(self.request.arguments) this gives me list of list. I don't know why? May be a general behaviour of Tornado. That why I access [0] for fields other than language. Because user can specify multiple language.